expense/expenseRouter.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+
+const Budgets = require('./expenseModel');
+
+const router = express.Router();
+
+router.post('/', (req, res) => {
+    const expenseInfo = req.body;
```

Comment:
you're very trusting of whats on the req.body.  Whoever is calling the route could technically put anything in there